server.js

```diff
@@ -299,7 +299,7 @@ app.get('/questionlist', function(req, res) {
     /* If this is the first time accessing it, fetch list from database. */
     if (req.session.adminQuestionList == null) {
         selector.findQuestions(0, selector.findTypes.SORT_DEFAULT,
-                                    null, function(questionlist) {
+                                    null, null, function(questionlist) {
```

Comment:
Things like this are worth putting some more comments. What are the nulls representing?

Reply:
The nulls represent when we want to show all questions. I'll add comments so this is more clear.